plugins/auth-keycloak/README.md

```diff
@@ -0,0 +1,68 @@
+# @amplication/plugin-auth-keycloak
+
+[![NPM Downloads](https://img.shields.io/npm/dt/@amplication/plugin-auth-keycloak)](https://www.npmjs.com/package/@amplication/plugin-auth-keycloak)
+
+Enable [KeyCloak](https://www.keycloak.org/) authentication on a service
+
+## Purpose
+
+This plugin adds the required code to use Passport Keycloak strategy on the generated NestJS application
+[**Keycloak**](https://www.keycloak.org/) is an authentication and authorization platform that provides the required tools to secure your applications and services.
+
+## Configuration
+This plugin requires the following settings during plugin configuration:
+- `KEYCLOAK_HOST`:   Complete URL of keycloak instance,
+- `KEYCLOAK_REALM`:   Name of the realm to be used for authentication (or the one you just created)
+- `KEYCLOAK_CLIENT_ID`:   Client ID of the client to be used for authentication (or the client we just created)
+- `KEYCLOAK_CLIENT_SECRET`:   Client Secret of the client to be used for authentication.
```

Comment:
These can only be provided after the keycloak is setup right?

Reply:
Right. These are manually created by Admin.
Keycloak allows admin to create multiple realms and in order to connect, these are required.

Reply:
But this is needed in plugin, so I suggest to let the users start up keycloak themselves by using a docker-compose file provided by you. Maybe can include the steps in README. Then they can provide these values to plugin.